contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -1125,6 +1130,26 @@ def test_aaf_marker_over_transition(self):
 
         self.assertIsNotNone(timeline)
 
+    def test_aaf_marker_over_audio_file(self):
+        """
+        Make sure we can transcibe markers over an audio AAF file.
+        """
+
+        timeline = None
+
+        try:
+            timeline = otio.adapters.read_from_file(
+                MARKER_OVER_AUDIO_PATH
+            )
+
+        except Exception as e:
+            print('[ERROR] Transcribing test sample data `{}` caused an exception: {}'.format(  # noqa
+                os.path.basename(MARKER_OVER_AUDIO_PATH),
+                e)
+            )
+
+        self.assertIsNotNone(timeline)
```

Comment:
Can you also check that the markers are in the right place (on the right item, and the right time)?

Reply:
Added